usr/lib/cca_stdll/cca_specific.c

```diff
@@ -692,10 +698,8 @@ uint16_t cca_inttok_privkey_get_len(CK_BYTE * tok)
 /* Given a CCA internal token private key object, get the modulus */
 CK_RV cca_inttok_privkey_get_n(CK_BYTE * tok, CK_ULONG * n_len, CK_BYTE * n)
```

Comment:
I would recommend to rename this 'n' to something more descriptive, maybe modulus ?

Reply:
Well, The CCA doc speaks about modulus n, so 'n' is not wrong. Although agree that 'modulus' would be a better name than 'n'. However, I do not want to mix this into this commit, since its not related.